### PR TITLE
New User Tabla

### PR DESCRIPTION
**Pull request by Alan-RM2899**
The new user table is already made so that they can give it a revised one

### CODE REVIEW

File: config/server.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const passport = require('./passport');
 
 const db = require('./db');
 require('../models/Customer.models');
```
Comment (DieGopherLT) [inline config/server.js:8]: I thought that customer were also gonna be included on user model.

File: models/User.models.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const { DataTypes } = require('sequelize');
+const db = require('../config/db');
+
+const User = db.define('User', {
+    UserID: {
+        type: DataTypes.INTEGER({length:10}),
+        primaryKey: true,
+        autoIncrement: true,
+        unique: true
```
Comment (DieGopherLT) [inline models/User.models.js:9]: Unique prop is not necessary in there, primaryKey prop will do the job.

**Review by DieGopherLT — COMMENTED** [review]
I left two observations to think about.

**Review by DieGopherLT — CHANGES_REQUESTED** [review]
The user properties start with a uppercase letter, in the rest of models they start with a lowercase one. Can you change that to keep the same code style?

**Comment by Alan-RM2899** [comment]
Change Request  complete

File: models/User.models.js [hunk 3]
```diff
@@ -1,48 +1,39 @@
...
+            msg: "Ese usuario ya existe"
+        }
+    },
+    password: {
         type: DataTypes.STRING(60),
         allowNull: false
     }
 }, {
     timestamps: false,
-    hooks: {
```
Comment (DieGopherLT) [inline models/User.models.js:37]: That is the part that allows to hash passwords, why did you delete it?

Comment (Alan-RM2899) [reply models/User.models.js:37]: Es para que tu la pongas por que yono puse eso asi que te toca a ti

Comment (DieGopherLT) [reply models/User.models.js:37]: Alright.
